Libplanet.Explorer.Executable/StoreRegistry.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Libplanet.Store;
+
+namespace Libplanet.Explorer.Executable
+{
+    internal class StoreRegistry
+    {
+        private static (string, Parser)[] stores = new (string, Parser)[]
```

Comment:
Can we omit `new (string, Parser)[]`?

Reply:
We can't omit here (according to the compiler).